spinalcordtoolbox/gui/widgets.py

```diff
@@ -139,13 +140,9 @@ def _init_ui(self, data, aspect):
         self._axes = self._fig.add_axes([0, 0, 1, 1], frameon=True)
         self._axes.axis('off')
         self.view = self._axes.imshow(
-            data,
+            self.adjust_image(data),
             cmap=self._params.cmap,
-            interpolation=self._params.interp,
-            vmin=self._params.vmin,
-            vmax=self._params.vmax,
-            alpha=self._params.alpha)
-        self._axes.set_aspect(aspect)
```

Comment:
Why removing it again?  It is needed for anisotropic resolution. I fixed this bug in a [recent commit](https://github.com/neuropoly/spinalcordtoolbox/pull/1545/commits/d9704a46d14fc53e16150708a3c57cd2c3f90f8e), and now you are reintroducing it without justification. 

Without self._axes.set_aspect(aspect):
![screen shot 2018-03-29 at 11 16 58](https://user-images.githubusercontent.com/2482071/38066498-2fc311aa-3343-11e8-8df6-b994292188f9.png)

With self._axes.set_aspect(aspect):
![screen shot 2018-03-29 at 11 18 25](https://user-images.githubusercontent.com/2482071/38066499-332722a0-3343-11e8-82a7-0537d8f7d753.png)